dockerfiles/Dockerfile-request

```diff
@@ -4,6 +4,10 @@ COPY pyproject.toml /root/swift_request/pyproject.toml
 COPY README.md /root/swift_request/README.md
 COPY swift_browser_ui /root/swift_request/swift_browser_ui
 
+# remove this folder as it creates issues with hatch
```

Comment:
Perhaps the comments should say that it's a link? It might be confusing to someone else who might read this later otherwise